code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -178,6 +196,78 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 49;
```

Comment:
This should be in common/constants, instead of being hardcoded (I think we already have a variable used by the RPC)

Reply:
Where is that located? I've seen that we're hardcoding this number many times throughout the codebase.
Happy to reference from a constants crate

Reply:
I think I am completely derping as I cannot find it.